mergify_engine/actions/rebase.py

```diff
@@ -59,6 +60,18 @@ def run(self, ctxt: context.Context, rule: rules.EvaluatedRule) -> check_api.Res
             if output:
                 return output
 
+            repo_info = ctxt.client.item(ctxt.pull["base"]["repo"]["url"])
+
+            if repo_info["size"] > config.NOSUB_MAX_REPO_SIZE_KB:
+                if not ctxt.subscription.has_feature(
+                    subscription.Features.LARGE_REPOSITORY
+                ):
```

Comment:
```suggestion
            if repo_info["size"] > config.NOSUB_MAX_REPO_SIZE_KB and not ctxt.subscription.has_feature(subscription.Features.LARGE_REPOSITORY):
```
no need for 2 if there